### PR TITLE
fix(store): follow good practices in store insert and query metrics

### PR DESCRIPTION
**Pull request by LNSD**
After the conversation with @jakubgs, some improvements have been introduced to the recently added metrics to adhere to [prometheus naming good practices](https://prometheus.io/docs/practices/naming/):

* Renamed the histogram metrics to `*_duration_seconds`
* Values are inserted in _seconds_ and not _milliseconds_

### CODE REVIEW

**Review by rymnc — APPROVED** [review]
LGTM

**Review by jakubgs — APPROVED** [review]
Thanks.

**Review by jm-clius — APPROVED** [review]
LGTM! I will add dashboards once this is merged and deployed.

**Pull request merged**
